### PR TITLE
PEPPER-171: dsm junit tests

### PR DESCRIPTION
**Pull request by SimoneMaiwald**
just some moving things around and fixing small things

### CODE REVIEW

File: pepper-apis/dsm-core/pom.xml [hunk 1]
```diff
@@ -275,30 +275,30 @@
...
-                org.broadinstitute.dsm.model.settings.field.FieldSettingsTest,
-                org.broadinstitute.dsm.model.ups.UPSActivityTest,
-                org.broadinstitute.dsm.pubsub.EditParticipantTest,
-                org.broadinstitute.dsm.pubsub.UpdateWorkflowStatusTest,
-                org.broadinstitute.dsm.RouteTest,
-                org.broadinstitute.dsm.route.familymember.AddFamilyMemberRouteTest,
-                org.broadinstitute.dsm.route.KitUploadRouteTest,
-                org.broadinstitute.dsm.KitStatusChangeRouteTest
-            </dsm.test.ci.excludes>
+        <dsm.test.excludes></dsm.test.excludes>
```
Comment (SimoneMaiwald) [inline pepper-apis/dsm-core/pom.xml:278]: ordered classes alphabetically

File: pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/DSMServer.java [hunk 2]
```diff
@@ -187,25 +187,23 @@ public class DSMServer {
...
-    public static final String GCP_PATH_TO_DSM_TASKS_SUB = "pubsub.dsm_tasks_subscription";
-    public static final String GCP_PATH_TO_DSM_TO_MERCURY_TOPIC = "pubsub.dsm_to_mercury_topic";
-    public static final String GCP_PATH_TO_MERCURY_TO_DSM_SUB = "pubsub.mercury_to_dsm_subscription";
-    private static final Logger logger = LoggerFactory.getLogger(DSMServer.class);
-    private static final String API_ROOT = "/ddp/";
-    private static final String UI_ROOT = "/ui/";
-    public static final String SIGNER = "org.broadinstitute.kdux";
-    private static final String[] CORS_HTTP_METHODS = new String[] {"GET", "PUT", "POST", "OPTIONS", "PATCH"};
-    private static final String[] CORS_HTTP_HEADERS =
+    private static final String gcpPathToPubsubProjectId = "pubsub.projectId";
```
Comment (SimoneMaiwald) [inline pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/DSMServer.java:190]: just renaming and settings things from public to private

File: pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/gbf/GBFOrderFinder.java [hunk 3]
```diff
@@ -56,7 +56,7 @@ public class GBFOrderFinder {
                     + "        and not exists (select 1 from ddp_participant_exit e where e.ddp_instance_id = i.ddp_instance_id "
                     + "and e.ddp_participant_id = untransmitted.ddp_participant_id) "
                     + "    union select distinct req.external_order_number, req.ddp_participant_id, req.ddp_instance_id, "
-                    + "      req.kit_type_id, req.dsm_kit_request_id rom ddp_kit_request req, ddp_instance i where  "
+                    + "      req.kit_type_id, req.dsm_kit_request_id from ddp_kit_request req, ddp_instance i where  "
```
Comment (SimoneMaiwald) [inline pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/gbf/GBFOrderFinder.java:59]: typo

File: pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/jobs/TestBostonUPSTrackingJob.java [hunk 4]
```diff
@@ -43,7 +43,7 @@
 import org.broadinstitute.dsm.statics.DBConstants;
 import org.broadinstitute.dsm.util.EventUtil;
 import org.broadinstitute.dsm.util.KitUtil;
-import org.broadinstitute.lddp.security.Auth0Util;
+import org.broadinstitute.dsm.security.Auth0Util;
```
Comment (SimoneMaiwald) [inline pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/jobs/TestBostonUPSTrackingJob.java:46]: moving of auth0Util

File: pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/AuthenticationRoute.java [hunk 5]
```diff
@@ -62,9 +45,8 @@ public class AuthenticationRoute implements Route {
     private final String audienceNameSpace;
 
     public AuthenticationRoute(@NonNull Auth0Util auth0Util, @NonNull UserUtil userUtil, @NonNull String auth0Domain,
-                               @NonNull String clientSecret,
-                               @NonNull String auth0ClientId, @NonNull String auth0MgmntAudience, @NonNull String audienceNameSpace) {
-
+                               @NonNull String clientSecret, @NonNull String auth0ClientId, @NonNull String auth0MgmntAudience,
```
Comment (SimoneMaiwald) [inline pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/AuthenticationRoute.java:48]: spacing

File: pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/AuthenticationRoute.java [hunk 6]
```diff
@@ -135,66 +115,6 @@ public Object handle(Request request, Response response) {
         return response;
     }
```
Comment (SimoneMaiwald) [inline pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/AuthenticationRoute.java:117]: moved stuff into auth0util

**Review by pegahtah — APPROVED** [review]

**Pull request merged**